Menus/Settings/Settings.cs

```diff
@@ -190,6 +190,12 @@ private void _on_diffculty_option_item_selected(int index)
 		}
 	}
 
+	// controls option function
+	private void _on_controls_option_item_selected(int index)
+	{
+		string selectedOption = ((OptionButton)GetNode("settings/gameplay_settings/controls_option")).GetItemText(index);
```

Comment:
Ideally we wouldn't be using GetNode, we should be using [Export] properties instead so we can set them in the editor

Reply:
I don't believe this is my area, might've accidentally changed something